docs/docs/releases/0.2.0-testing.1.md

```diff
@@ -0,0 +1,234 @@
+# 🧪 Mathesar 0.2.0-testing.1
+
+!!! danger "Not a stable release"
+    This is a testing build released with the goal of gathering feedback from our community. It has **[many known issues](https://github.com/mathesar-foundation/mathesar/issues?q=is%3Aissue+milestone%3Av0.2.0-testing.2+)** and is not recommended for production use.
+
+## Summary
+
+**Mathesar 0.2.0-testing.1** provides an early preview of improvements we plan to release in our beta version. We have a brand new access control system based entirely on PostgreSQL database permissions, and we've also made major improvements to the responsiveness of the Mathesar UI. We've also built out a new RPC API, and we're deprecating our REST API in favor of it.
+
+_This page provides a comprehensive list of all changes in the release._
+
+## Installing 0.2.0-testing.1
+
+You will need to create a new installation of Mathesar to use this version, which you can do via [Docker Compose](../installation/docker-compose/index.md) or [from source](../installation/build-from-source/index.md).
```

Comment:
Docker compose currently downloads from the prod image. We would have to update it. The instructions in these pages need to target the testing branch.

Reply:
Fixed in https://github.com/mathesar-foundation/mathesar/commit/172f5e7e3762bff408be7c9aafb22fdb5827cf4f

Reply:
Thank you!